core/test/utilities/helpers/escaping-helpers.spec.js

```diff
@@ -1,13 +1,21 @@
 const chai = require('chai');
 const assert = chai.assert;
 const sinon = require('sinon');
-const EscapingHelpers = require('../../../src/utilities/helpers/escaping-helpers');
+const EscapingHelpers = require('../../../src/utilities/helpers/escaping-helpers')
```

Comment:
Indeed it works, I used to remember that it never worked with es6 import. Maybe because of those older node versions, lets see if it breaks some time again.

Reply:
Please remember to change this one ;)